airbyte-webapp/src/views/Connection/ConnectionForm/ConnectionForm.tsx

```diff
@@ -113,13 +112,12 @@ interface ConnectionFormProps {
   className?: string;
   additionBottomControls?: React.ReactNode;
   successMessage?: React.ReactNode;
-  onReset?: (connectionId?: string) => void;
   onDropDownSelect?: (item: DropDownRow.IDataItem) => void;
   onCancel?: () => void;
   onChangeValues?: (values: FormikConnectionFormValues) => void;
 
   /** Should be passed when connection is updated with withRefreshCatalog flag */
-  editSchemeMode?: boolean;
+  allowSavingUntouchedForm?: boolean;
```

Comment:
`canSubmitUntouchedForm`? This component has no use of the word "save" but instead it cares about submit